app/controllers/data_objects_controller.rb

```diff
@@ -591,4 +594,14 @@ def visibility_from_params(he)
     params["visibility_id_#{he.id}"] ? Visibility.find(params["visibility_id_#{he.id}"]) : nil
   end
 
+  def limit_data_objects
+    if current_user.newish? || (current_user.newish? && current_user.assistant_curator?)
+       articles =  current_user.data_objects.where(data_type_id: DataType.text.id,
+                    created_at: DateTime.now.to_date.beginning_of_day..DateTime.now.to_date.end_of_day)
```

Comment:
Line is too long. [103/80]<br>Align the elements of a hash literal if they span more than one line.<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.